src/components/molecules/Darkmode/Darkmode.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { Icon } from 'atoms'
+
+const Darkmode = ({ isActive, toggle, cssStyles, isSupported }) => {
+  if(!isSupported) {
+    return null
+  }
+
```

Comment:
@Joe-Pena We have an HOC method to handle this sort of logic. Check out how we are using `withMaybe` in other components. It will basically render `null` if the callback you pass in returns false. Here's an example that would go in the container. Again, I'm not sure if isSupported would be the prop to use here because it is never being called, but you get the idea. I would also place this handler near the bottom of your container so it is getting all the props it needs.

```
const nullConditionFn = ({ isSupported }) => !isSupported

export default compose(
  withMaybe(nullConditionFn),
  ...
```